### PR TITLE
Base fingerprinting 71 versioned list for Firefox-iOS

### PR DESCRIPTION
**Pull request by skim1102**
# About this PR
After auditing tracking protections lists used in the mobile apps I found a discrepancy in the fingerprinting lists. In order to maintain a more uniform tracking protections lists across mobile and desktop apps, this PR provided JSON formatted fingerprinting tracking protection list.

#
This PR blocks https://github.com/mozilla-mobile/firefox-ios/issues/5559

### CODE REVIEW

File: firefox-ios/base-fingerprinting-track.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
+[
```
Comment (skim1102) [inline firefox-ios/base-fingerprinting-track.json]: @englehardt, should I use a more broader name for the folder like `normalized-lists` instead?

Comment (englehardt) [reply firefox-ios/base-fingerprinting-track.json]: Yes let's do that -- in case we have other mobile products using the lists in the future.

**Comment by englehardt** [comment]
How is this list generated? Have you uploaded that code anywhere?

**Comment by skim1102** [comment]
@englehardt I do not have the code uploaded anywhere yet and I generated this list from the code I have locally. I will likely include it in https://github.com/mozilla-services/shavar-list-creation/pull/100, but for now I wanted to have a placeholder lists that the Firefox iOS team can use to test their functionalities.

**Comment by skim1102** [comment]
@englehardt I did some analysis on the difference between the master list to the versioned lists for "base-fingerprinting-track-digest256" but couldn't find `cbox.ws` in any. Also, the domains you requested to be added are from the master list (73 and newer) and since we want the mobile apps to use the latest release those domains should not be included in the 72 and 71 versioned lists, right?

**Comment by englehardt** [comment]
> @englehardt I did some analysis on the difference between the master list to the versioned lists for "base-fingerprinting-track-digest256" but couldn't find `cbox.ws` in any.

It's under the [`Content` category](https://github.com/mozilla-services/shavar-prod-lists/blob/0126789ddb987cef3a48e6d60a06cfa52ba64990/disconnect-blacklist.json#L7413-L7418) and the [`Fingerprinting` category](https://github.com/mozilla-services/shavar-prod-lists/blob/0126789ddb987cef3a48e6d60a06cfa52ba64990/disconnect-blacklist.json#L10061-L10066), so it should be [on the list](https://github.com/mozilla-services/shavar-list-creation-config/blob/master/prod.ini#L204). I see it in `about:url-classifier`.

> Also, the domains you requested to be added are from the master list (73 and newer) and since we want the mobile apps to use the latest release those domains should not be included in the 72 and 71 versioned lists, right?

Yes, with the exception of `cbox.ws`. And I agree that the mobile apps should use whatever is the latest desktop release version at the time of release.

**Pull request merged**